### PR TITLE
libct/isPaused: don't use GetPaths from v2 code

### PR DESCRIPTION
**Pull request by kolyshkin**
Using `GetPaths` from cgroupv2 unified hierarchy code is deprecated
and this function will (hopefully) be removed.

Use `GetUnifiedPath()` for v2 case.

There are more cases like this that we still need to fix, but this one is a low hanging fruit :)

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

File: libcontainer/container_linux.go [hunk 1]
```diff
@@ -1822,14 +1822,23 @@ func (c *linuxContainer) runType() Status {
...
-	if cgroups.IsCgroup2UnifiedMode() {
+	var fcg, filename, pausedState string
+
+	if !cgroups.IsCgroup2UnifiedMode() {
+		fcg = c.cgroupManager.GetPaths()["freezer"]
+		if fcg == "" {
+			// A container doesn't have a freezer cgroup
+			return false, nil
+		}
+		filename = "freezer.state"
```
Comment (tedyu) [inline libcontainer/container_linux.go:1833]: Just curious: the filenames are different for the unified versus non-unified cases. I assume this is intentional.

Comment (kolyshkin) [reply libcontainer/container_linux.go:1833]: @tedyu yes, cgroups v2 have different filenames for various controllers.

**Pull request merged**